packages/dapp/src/components/Navbar/Menu/ConnectedMenu/ConnectedMenu.tsx

```diff
@@ -1,55 +1,161 @@
+  }, [account]);
+
+  const handleCloseSnackbar = (
+    event?: React.SyntheticEvent | Event,
+    reason?: string,
+  ) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setCopiedToClipboard(false);
+  };
 
-  return !!menu.openNavbarConnectedMenu ? ( //todo, ON ???
+  return !!menu.openNavbarConnectedMenu ? (
     <NavbarMenu
-      handleClose={handleClose}
-      open={menu.openNavbarConnectedMenu}
-      isScrollable={menu.openNavbarSubMenu === SubMenuKeys.chains}
+      open={true}
+      isScrollable={true}
       setOpen={menu.onOpenNavbarConnectedMenu}
+      handleClose={handleClose}
       isOpenSubMenu={menu.openNavbarSubMenu !== SubMenuKeys.none}
     >
-      {_connectedMenuItems.map((el, index) => (
-        <MenuItemComponent
-          key={`${el.label}-${index}`}
-          label={el.label}
-          prefixIcon={el.prefixIcon}
-          showMoreIcon={el.showMoreIcon}
-          triggerSubMenu={el.triggerSubMenu}
-          showButton={el.showButton}
-          suffixIcon={el.suffixIcon}
-          onClick={el.onClick}
-          open={menu.openNavbarConnectedMenu}
-          isOpenSubMenu={menu.openNavbarSubMenu !== SubMenuKeys.none}
-          setOpenSubMenu={menu.onOpenNavbarSubMenu}
+      <Box textAlign={'center'} mt={theme.spacing(3)}>
+        <Avatar
+          src={walletIcon}
+          // alt={`${!!usedWallet.name ? usedWallet.name : ''}wallet-logo`}
+          sx={{
+            padding: theme.spacing(4.5),
+            background:
+              theme.palette.mode === 'light'
+                ? theme.palette.black.main
+                : theme.palette.white.main,
+            margin: 'auto',
+            height: '96px',
+            width: '96px',
+          }}
         />
-      ))}
+        <Typography variant="lifiBodyLargeStrong" mt={theme.spacing(4)}>
+          {_walletDigest}
+        </Typography>
+      </Box>
+      <Box
+        display={'flex'}
+        flexDirection={'row'}
+        justifyContent={'space-between'}
+        sx={{
+          margin: `${theme.spacing(3)} auto ${theme.spacing(2)}`,
+          width: '75%',
+          minWidth: '280px',
 
-      <SubMenuComponent
-        label={`${translate(`${i18Path}chains`)}`}
-        isSubMenu={true}
-        isScrollable={true}
-        triggerSubMenu={SubMenuKeys.chains}
-        open={menu.openNavbarConnectedMenu}
-        isOpenSubMenu={menu.openNavbarSubMenu !== SubMenuKeys.none}
-        setOpenSubMenu={menu.onOpenNavbarSubMenu}
-        subMenuList={_connectedSubMenuChains}
-      />
+          [theme.breakpoints.up('sm' as Breakpoint)]: {
+            marginTop: theme.spacing(6),
+            width: 'auto',
+            minWidth: 'inherit',
+          },
+        }}
+      >
+        <SpotButton
+          name="Copy"
+          onClick={() => {
+            navigator?.clipboard?.writeText(account.address);
+            setCopiedToClipboard(true);
+            trackEvent({
+              category: 'menu',
```

Comment:
For tracking fields,
`category`, `action` and `labels`; do we have events defined in a `type` or we are using strings?

if it's just typed as string then there is a possible case of typo in future which could affect the analytics

Reply:
That´s a great point! I have not thought of that, but adding some kind of object with some pre-defined strings makes a lot of sense!